### PR TITLE
Add "Principal Technical Consultant"

### PR DESCRIPTION
**Pull request by lukemorton**

### CODE REVIEW

File: roles/principal_technical_consultant.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Principal Technical Consultant
+
+Our Principal Technical Consultants define and drive strategy to deliver digital and technology outcomes for our customers. Working with senior customer stakeholders our consultants drive digital transformation.
```
Comment (RoryMacDonald) [inline roles/principal_technical_consultant.md]: I'm unsure about the digital transformation angle we've taken here and further down the document. Feel too broad in scope and should limit around technology strategy (cloud, devops, software engineering, agile etc)

File: roles/principal_technical_consultant.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+- Our handbook is open sourced
+- We are vegan and non-drinker friendly as well as meat-eater and drinker friendly
+- Retreats and trips every year
+
+## Salary
+
+This role has a salary of £85-95k depending on experience.
+
+
+## Expectations
```
Comment (RoryMacDonald) [inline roles/principal_technical_consultant.md]: Should there be clearer expectations around 'buy' ecosystem? I would expect somebody in this role to be able to provide impartial advice around options. e.g. Salesforce is an option, as it provides X, Y and Z results.

Comment (RoryMacDonald) [inline roles/principal_technical_consultant.md]: Should we be more specific around expectations and areas we expect? Worry we're getting a bit too fluffy.

Agile Operating Models - How to set up an organisation and its teams for success. Looking at current practices and what needs 

Cloud - Exposure to cloud providers such as Azure, AWS, GCP. Can talk about migration strategies, business benefits of each. 

Devops - Understanding of PaaS, IaaS and SaaS, business benefits. Talk about infrastructure as code, Devops ways of working, CI and CD.    

Architecture - Ability to take a business problem and produce a high-level architecture to enable it. Make sensible decisions around build vs buy, integration strategies etc. 

Buy - Research and evaluable the market for architectural components. e.g. Look at Pega vs Open Source case management. Prepare and present options. 

Software Engineering - Understanding modern software engineering (TDD, BDD, Clean Code, DDD) etc. 

Data - High level thinking about the architectural impact on data and relevant regulations (e.g. GDPR). Ability to talk about data engineering practices (pipelines, Hadoop etc)

**Comment by cob16** [comment]
@lukemorton Can this be closed now? The role was removed in #218

**Pull request closed without merging**